vshard/consts.lua

```diff
@@ -9,6 +9,11 @@ return {
         GARBAGE = 'garbage',
     },
 
+    BUCKET_TRIGGER = {
+      RECV = 'bucket_data_recv_txn',
+      GC = 'bucket_data_gc_txn'
```

Comment:
Please, fix the alignment (you used 2-space indentation, but it should be 4-space). And rename `BUCKE_TRIGGER` to `BUCKET_EVENT`.

Reply:
Fixed